d3a_api_client/utils.py

```diff
@@ -166,6 +167,19 @@ def get_area_uuid_and_name_mapping_from_simulation_id(collab_id, domain_name):
         return area_name_uuid_map
 
 
+def get_aggregators_list(domain_name):
```

Comment:
IMO the domain_name should be optional here and the global environmental setting should be used as default.